.github/workflows/analyze.yml

```diff
@@ -18,9 +16,9 @@ jobs:
       - name: Install pub dependencies
         run: flutter/bin/flutter pub get
       - name: Verify formatting
-        run: flutter/bin/dart format --output=none --set-exit-if-changed lib test
+        run: flutter/bin/dart format --output=none --set-exit-if-changed lib test packages
       - name: Analyze project source
-        run: flutter/bin/dart analyze --fatal-infos lib
+        run: flutter/bin/dart analyze --fatal-infos lib test
```

Comment:
It's very tricky to run analysis on the packages directory so I decided to verify only formatting.